lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -410,7 +410,10 @@ def _section_course_info(course, access):
         'start_date': course.start,
         'end_date': course.end,
         'num_sections': len(course.children),
-        'list_instructor_tasks_url': reverse('list_instructor_tasks', kwargs={'course_id': six.text_type(course_key)}),
+        'list_instructor_tasks_url': reverse(
+            'api_instructor:list_instructor_tasks',
+            kwargs={'course_id': six.text_type(course_key)}
```

Comment:
No longer need to use six

Reply:
Please avoid making unrelated changes. You only needed to not use six in the code being changed in this PR.

Reply:
I've reverted otherwise unimpacted lines of code